helm/cluster/files/etc/kubelet-configuration.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+shutdownGracePeriod: 300s
+shutdownGracePeriodCriticalPods: 60s
+kernelMemcgNotification: true
+evictionSoft:
+  memory.available: "500Mi"
+evictionHard:
+  memory.available: "200Mi"
+  imagefs.available: "15%"
+evictionSoftGracePeriod:
+  memory.available: "5s"
+evictionMaxPodGracePeriod: 60
+kubeReserved:
+  cpu: 350m
+  memory: 1280Mi
+  ephemeral-storage: 1024Mi
+kubeReservedCgroup: /kubereserved.slice
+protectKernelDefaults: true
+systemReserved:
+  cpu: 250m
+  memory: 384Mi
+systemReservedCgroup: /system.slice
+tlsCipherSuites:
+- TLS_AES_128_GCM_SHA256
```

Comment:
Compared to cluster-aws change, I have extended this list so we include all preferred TLS cipher suites that are listed in official [kubelet docs](https://v1-25.docs.kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) (for CLI flag `--tls-cipher-suites`, but same values apply for `KubeletConfiguration`).